azurerm/resource_arm_subnet.go

```diff
@@ -111,7 +111,8 @@ func resourceArmSubnet() *schema.Resource {
 										Elem: &schema.Schema{
 											Type: schema.TypeString,
 											ValidateFunc: validation.StringInSlice([]string{
-												"Microsoft.Network/virtualNetworks/subnets/action",
```

Comment:
is this value no longer valid?

Reply:
this still appears to be a valid value: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/VirtualNetworkGetWithServiceAssociationLink.json#L56 - as such can we add this back in?